### PR TITLE
added packages/minicli/minicli.5.0.2

### PR DESCRIPTION
**Pull request by UnixJunkie**
added dune-project file in the source tree

### CODE REVIEW

File: packages/minicli/minicli.5.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage: "https://github.com/UnixJunkie/minicli"
+bug-reports: "https://github.com/UnixJunkie/minicli/issues"
+dev-repo: "git+https://github.com/UnixJunkie/minicli.git"
+build: [
+  ["dune" "build" "-p" name]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/minicli/minicli.5.0.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
According to your `dune-project` file, this package requires at least dune 1.6

File: packages/minicli/minicli.5.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+homepage: "https://github.com/UnixJunkie/minicli"
+bug-reports: "https://github.com/UnixJunkie/minicli/issues"
+dev-repo: "git+https://github.com/UnixJunkie/minicli.git"
+build: [
+  ["dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/minicli/minicli.5.0.2/opam]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

File: packages/minicli/minicli.5.0.2/opam [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ["dune" "runtest" "-p" name] {with-test}
```
Comment (kit-ty-kate) [inline packages/minicli/minicli.5.0.2/opam]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

**Comment by UnixJunkie** [comment]
thanks for the proposed fixes

**Comment by UnixJunkie** [comment]
should be OK to merge

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**